docs/tutorials/new-feature-tutorial.md

```diff
@@ -507,66 +508,78 @@ function _setup_page() {
 The JavaScript code for organization settings and permissions can be found in
 `static/js/settings_org.js`.
 
-There is a front-end version of `property_types`, which reduces the code
-needed on the front end for a new feature.
+In frontend, we have split the `property_types` into three objects viz.
 
-Add the new feature to the `property_types` object in `settings_org.js`.
-The key should be the setting name and the value should be an object with
-the following keys:
+- `org_profile`: This contains properties which belong to organization profile
+    section. New features don't usually lie in this section as it mostly contains
+    the bio of the realm.
 
-* type
-* checked_msg (what message the user sees when they enable the setting)
-* unchecked_msg (what message the user sees when they disable the setting)
+- `org_settings`: This contains properties which belong to organization
+settings section. Settings belonging to this section generally decide what
+features should be available to a user like deleting a message, message
+edit history etc. and usually they affect the whole realm including admin
+(who definitely have access to change them). Our `mandatory_topics`
+feature is going to lie in this section.
+
+- `org_permissions`: This contains properties which belong to organization
+permissions section. These properties are related to the user access to
+the various feature like `bot_creation_policy` which decide who can create bots.
+
+After deciding the section of setting, you've to find the appropriate subsection of the new property
```

Comment:
"section of setting" is poor grammar; we want something like "After deciding which section the setting belongs in..."